hail_scripts/reference_data/config.py

```diff
@@ -106,6 +138,28 @@ def custom_mpc_select(ht):
             'select': ['PHRED'],
         },
     },
+    'clinvar': {
+        '37': {
+            'custom_import': download_and_import_latest_clinvar_vcf,
+            'path': 'ftp://ftp.ncbi.nlm.nih.gov/pub/clinvar/vcf_GRCh37/clinvar.vcf.gz',
+            'select': {'alleleId': 'info.ALLELEID'},
+            'custom_select': clinvar_custom_select,
+            'enum_select': {
+                'pathogenicity': CLINVAR_PATHOGENICITIES,
+                'assertion': CLINVAR_ASSERTIONS,
+            },
+        },
+        '38': {
```

Comment:
Out of scope for MVP, but at some point its probably worth looking into the fact that for many of these the configs 37 and 38 are the same except for the file path.